google/cloud/bigtable/internal/instance_admin.cc

```diff
@@ -234,6 +234,86 @@ void InstanceAdmin::DeleteAppProfile(bigtable::InstanceId const& instance_id,
       "InstanceAdmin::DeleteAppProfile", status);
 }
 
+::google::iam::v1::Policy InstanceAdmin::GetIamPolicy(
+    std::string const& resource, grpc::Status& status) {
+  auto rpc_policy = rpc_retry_policy_->clone();
+  auto backoff_policy = rpc_backoff_policy_->clone();
+
+  ::google::iam::v1::GetIamPolicyRequest request;
+  request.set_resource(resource);
+
+  MetadataUpdatePolicy metadata_update_policy(project_name(),
+                                              MetadataParamTypes::RESOURCE);
+
+  return ClientUtils::MakeCall(*client_, *rpc_policy, *backoff_policy,
+                               metadata_update_policy,
+                               &InstanceAdminClient::GetIamPolicy, request,
+                               "InstanceAdmin::GetIamPolicy", status, true);
+}
+
+::google::iam::v1::Policy InstanceAdmin::SetIamPolicy(
+    std::string const& resource, std::int32_t const& version,
+    google::cloud::IamBindings const& iam_bindings, std::string const& etag,
+    grpc::Status& status) {
+  auto rpc_policy = rpc_retry_policy_->clone();
+  auto backoff_policy = rpc_backoff_policy_->clone();
+
+  ::google::iam::v1::Policy policy;
+  policy.set_version(version);
+  policy.set_etag(etag);
+  auto role_bindings = iam_bindings.bindings();
+  for (auto& binding : role_bindings) {
+    auto binding_to_be_added = policy.add_bindings();
```

Comment:
maybe s/binding_to_be_added/new_binding/ or `binding_in_proto`?  When I read `binding_to_be_added` I immediately thought this was a new binding to add on the server side, and it is not that.